Optimize PageHeader,title and subtitle size for smaller screens.

UtkarshKr007 opened the pull request:
Additionally remove parameter from PageHeader component to specify large headers.
closes #1

UtkarshKr007 committed:
Optimize PageHeader,title and subtitle size for smaller screens.

Additionally remove parameter from PageHeader component to specify large headers.
closes #1

michael-small commented:
On first sweep, looking at the end result and not the code: looks great on most mobile screens (except the iPhone below), but perhaps the header can be a little bigger on desktops? 

iPhone 5/SE (320x568) 
![image](https://user-images.githubusercontent.com/33669563/82576686-956ea700-9b4f-11ea-950b-85d823181fa7.png)
I think this is because it's the narrowest width screen. Do you think it would be worth it to have a media query for small widths?

Also, could we perhaps crop it a bit less on desktop?
![image](https://user-images.githubusercontent.com/33669563/82576871-d6ff5200-9b4f-11ea-8593-3f537833a08a.png)
Or at the very least have it center a bit different on desktop?

I can impliment any of these changes but I want to make sure we have some agreement.

UtkarshKr007 committed:
Add media query for screens with width <350

Additionally, change class to className in popup.js, because ReactJS uses JSX(which uses className) rather than HTML.

Commit:
Merge branch 'master' into PageHeader

michael-small reviewed (commented):
Looks good to me. Perhaps we could just have an alternate photo for desktop, but it's fine now as is.
